packages/UsageAnalysis/files/Test Track/Bio/MSA.md

```diff
@@ -0,0 +1,13 @@
+1. Open sample_FASTA.csv.
+2. Add a new column with a formula RandBetween(0,5) - this will be the clusters column.
+3. On the menu ribbon, open **Bio** > **MSA**.
+4. Set Cluster to the new column with a formula (from the step 2).
+5. Check that **Alignment parameters** button adds input parameters to the dialog properly   
+6. Check the new column.
+
+* Everything is good if the new MSA column is added, the renderer for the column is set, and the sequences within a cluster are of the same length.
```

Comment:
> sequences within a cluster are of the same length.
How to check it?

Reply:
@ssSavenko FYI

Reply:
It means that it has right markup, and html elements dont have any visual conflicts

Reply:
it does not tell me how to check it
what does it mean 'same length'?